### DISCUSSION TITLE
Loading gcode files

### DISCUSSION THREAD

**Discussion by vphilli** [winder/Universal-G-Code-Sender#2264] [Help]
When I try to load a gcode file I get a response - Cannot find registration for mime type "text(x)gcode". 
I'm trying to find out why I get the message and how to fix it. The file doesn't load. I've tried this with several files.

Thanks

**Comment by breiler** [comment]
Could you attach the log file: https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#log-files

**Comment by vphilli** [comment]
Thanks for replying. I reinstalled UGS and it's working now! I didn't see where to get the log file. 





On Thursday, July 20, 2023 at 12:41:08 AM CDT, Joacim Breiler ***@***.***> wrote: 







Could you attach the log file: https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#log-files
